specifyweb/frontend/js_src/lib/components/Reports/index.tsx

```diff
@@ -135,7 +137,14 @@ export function ReportsView({
       >
         <div className="flex flex-col gap-4">
           <section>
-            <h2>{reportsText.reports()}</h2>
+            <div className="flex items-center gap-2">
+              <h2>{reportsText.reports()}</h2>
+              <img
```

Comment:
In the future, use `<TableIcon name="Reports" label={false} />` instead of the raw img tag